lib/MongoDB/Op/_BulkWrite.pm

```diff
@@ -223,9 +278,10 @@ sub _execute_write_command_batch {
                     );
                 }
                 else {
-                    unshift @left_to_send, $self->_split_chunk( $link, $chunk, $_->size );
+                    unshift @left_to_send, $self->_split_chunk( $chunk, $_->size );
                 }
             }
+            # Put retryable writes catches... no not here... damn... ermm...  oh. Could increment transaction id at the beginning before the loop, then after the redo increment again. Then it wont get caught? hmm.... food
```

Comment:
?!?

Reply:
Ah oops I thought i got rid of that comment... Was from the odd debugging stuff when I tied the driver in knots with $link